### ISSUE TITLE
PKI

### ISSUE THREAD

**Issue by zimbonz** [MicrosoftDocs/windows-itpro-docs#1412]
The instructions around PKI are a little vague. 
Can you please clarify what are the actual requirements for a server 2016 configuration of WH4B? DO we need to specifically follow the exhaustive instructions of configuring a 2 tier PKI , in order to get wh4b to work?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e0e839ae-7beb-07fe-f0ea-560079f1f858
* Version Independent ID: af8e8903-fd5c-7210-768d-e0b5de67fe88
* Content: [Validate Public Key Infrastructure (Windows Hello for Business)](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-pki)
* Content Source: [windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-pki.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-pki.md)
* Product: **w10**
* GitHub Login: @DaniHalfin
* Microsoft Alias: **daniha**

**Comment by egallagh** [comment]
It's not just a matter of getting it to work. It is a matter of getting it to work as intended. In order to implement a PKI "properly" you need at least two tiers.
So I guess it depends on what we mean by "properly". In this case the whole reason behind WH4B at its heart is security. If you implement a single CA then any compromise of that system compromises the entire certificate chain. You security is gone. Possibly worse than gone because you think you are still protected.
So ultimately, do you need more than one tier to get it to "work"? No, one will work.
Will doing so accomplish the goal of helping to secure your systems? No. No way. Absolutely no.
Hope that helps a little.

**Comment by DaniHalfin** [comment]
To clarify - there are multiple different ways to deploy PKI in your environment. The method described here is a support method to deploy PKI that will work correctly with WHfB. 
There are other ways to deploy PKI that will still leave your with a working solution, but aren't necessarily supported. It's hard to cover every possible method of deployment for WHfB and it's dependencies, so we've chosen to focus on the best practices. 
If you have specific questions regarding your environment, please reach out to our support channels, as they are, for sure, better equipped to assist you.

Thanks for reaching out, @zimbonz .

**Comment by DaniHalfin** [comment]
@shortpatti - please close this issue.
If there are any updates to the original feedback, please reply back. Thanks!

**Comment by zimbonz** [comment]
I may have not been clear - let me try again.
Is this page, providing detail for setting up a single tier, PKI setup - necessary for "testing" Wh4B? 

If so, then the  The section above with the following text, should be separated out with the additional descriptor, for example:
If you want to perform this in a production environment, according to best practices, then please follow these links instead:

"If you do have an existing public key infrastructure, please review Certification Authority Guidance from Microsoft TechNet to properly design your infrastructure. Then, consult the Test Lab Guide: Deploying an AD CS Two-Tier PKI Hierarchy for instructions on how to configure your public key infrastructure using the information from your design session."

What is unclear, is whether the links for 2-tier configuration are a pre-requisite of the instructions in this page or not.

**Comment by DaniHalfin** [comment]
Hi @zimbonz - we state the prerequisites [right here](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-identity-verification#on-premises-deployments). 
The quoted part of the guide is meant to provide general guidance. As stated, it is advisable to read the CA design guidance first and consult the test lab guide second to configure your PKI setup. We provide this information, as the deployment guide can't cover the full scope of PKI configuration, but some people might still need guidance in that area. This guide is only intended to cover what is needed to setup a successful WHfB deployment.

In general, if something is a requirement, we clearly state that. If you find it to be otherwise, please let us know, so that we can correct that.
